services/influxdb/service.go

```diff
@@ -144,22 +145,25 @@ func (s *Service) NewNamedClient(name string) (client.Client, error) {
 }
 
 type influxdb struct {
-	configs        []client.HTTPConfig
-	i              int
-	configSubs     map[subEntry]bool
-	exConfigSubs   map[subEntry]bool
-	hostname       string
-	httpPort       int
-	logger         *log.Logger
-	protocol       string
-	udpBind        string
-	udpBuffer      int
-	udpReadBuffer  int
-	startupTimeout time.Duration
-	disableSubs    bool
-
-	clusterID string
-	subName   string
+	configs                  []client.HTTPConfig
+	i                        int
+	configSubs               map[subEntry]bool
+	exConfigSubs             map[subEntry]bool
+	hostname                 string
+	httpPort                 int
+	logger                   *log.Logger
+	protocol                 string
+	udpBind                  string
+	udpBuffer                int
+	udpReadBuffer            int
+	startupTimeout           time.Duration
+	subscriptionSyncInterval time.Duration
+	disableSubs              bool
+	runningSubs              map[subEntry]bool
```

Comment:
This map needs to be initialized or you get a `panic: assignment to entry in nil map`.